src/platform/site-wide/sass/merger.scss

```diff
@@ -549,12 +551,28 @@
       a.usa-current {
         font-weight: bold;
         margin-left: 0;
+        padding-left: 25px;
       }
     }
 
     .usa-sidenav-sub_list{
       a {
-        width: 100% !important; // sorry this is temporary
+        width: 98% !important; // sorry this is temporary
```

Comment:
Why did this go down to 98%?

Reply:
This was done because the focus state was getting cut off.
<img width="1177" alt="screen shot 2018-10-15 at 8 17 20 am" src="https://user-images.githubusercontent.com/5377648/46962339-bc9df200-d057-11e8-8730-047fd65cd470.png">

Reply:
The border on focus was getting cut off on the right side, so I had to reduce it for it to show correctly.